EU4toV2/Data_Files/blankMod/output/common/cultures.txt

```diff
@@ -615,27 +615,27 @@ italian = {
 	laessin = { 
 		color = { 214 156 129 }
 
-		first_names = { Abucatui Achemnatui Acheporecui Adrian Albinui Albui Allepsandri Ambarui Ambro�ui Anaronacui Anasta�ui
-			Andriai A�echui A�erestui Anto�inui Anto�ui A�ui Aporo�ui Arcadzui Archemnui Artapsiai Auetui Augstinui
-			Augstui Aurelleanui Aurellui Bartoromeui Ba�ellui Bechellui Bellesariui Be�acui Be�edeptui Betumnatui
-			Bra�od�enui Bre�ui Breta�ecui Brezui Buld�ui Camurod�enui Carinui Carui Ca�ui Centui Ch�chellui Ch�llestinui
-			Cheprianui Cher�i Ch�sa Chetsiui Chetui Chodo�ui Chuspai Ciaudzui Ciementsui Cometai Consta�i Constantinui
-			Constantsui Cor�ellui Crasui Dadacui Dariui Deca�ui Dechemui Dechui Dentupei Desachentui Domechanui Dome�ecui
-			Dometsui Drusui D�era�ui D�ermanui D�esatorecui D��ui �d�edzui Ellanui Elpedzui �melleanui �mellui Emnatsui
-			E�odzui Eposomnatui Eptapori Ercullui Esarnui �tsui Eubarnacui Fabeanui Fabiui Fabrichui Faran�i Faustui
-			Fellepui Felli Flaueanui Floreanui Gallenui Galleriui Garui Gordeanui Grachanui Gracui Ioa�i Ioueanui
-			Iulleanui Iullui Iu�ui Iusche�eanui Iustinui Laurentsui Lleberiui Lleche�ui Llo Llontsui Luchui Madacui
+		first_names = { Abucatui Achemnatui Acheporecui Adrian Albinui Albui Allepsandri Ambarui Ambroui Anaronacui Anastaui

```

Comment:
Starting about here, a lot of characters start changing in the names. Was that intentional?

Reply:
Absolutely not, but Github doesn't seem to like anything outside Basic Latin

Reply:
Got it. I think I can put together a fix, but it's late so it'll have to be tomorrow or Saturday.